src/octoprint/templates/tabs/gcodeviewer.jinja2

```diff
@@ -3,6 +3,16 @@
         <input id="gcode_slider_layers" type="text">
         <canvas id="gcode_canvas" width="568" height="568"></canvas>
         <input id="gcode_slider_commands" type="text" style="width: 554px">
+        <div id ="button_control" class="btn-group-toolbar" style="width: 100%; padding-bottom: 7px; padding-top: 5px;text-align: center;">
+            <button id = "btn_layer_up" type="button" class="btn btn-primary btn-medium" style="width:48%" disabled='disabled'>
+                <i class="icon-white icon-arrow-up"></i>
+                <span>Layer Up</span>
```

Comment:
The `Layer Up` text should be translatable. All you need to do for that to work is wrap the text in `{{ _('...') }}`, so `{{ _('Layer up') }}` and `{{ _('Layer down') }}` further below.